src/app/public/modules/back-to-top/back-to-top.component.scss

```diff
@@ -0,0 +1,13 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+.sky-back-to-top {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  margin-top: $sky-margin-double;
```

Comment:
Do we need a margin-top if the container is getting added to the dock?

Reply:
Nope. Good catch!